keras_retinanet/bin/train.py

```diff
@@ -39,25 +39,34 @@
 from ..preprocessing.pascal_voc import PascalVocGenerator
 from ..preprocessing.csv_generator import CSVGenerator
 from ..preprocessing.open_images import OpenImagesGenerator
-from ..models.resnet import resnet50_retinanet, custom_objects
+from ..models.resnet import resnet_retinanet, custom_objects, download_imagenet
 from ..utils.transform import random_transform_generator
 from ..utils.keras_version import check_keras_version
 
 
+RESNET_BACKBONE = 50
```

Comment:
Maybe we should make this an argument?

Reply:
We could do that. I'd prefer to do that in a separate PR though, and keep this one focussed on weights loading.

Reply:
Alright, then this LGTM :)